src/app/Authenticated.tsx

```diff
@@ -72,7 +72,8 @@ export const authenticatedRoutes = {
             },
             params: {
                 connectorID: 'connectorID',
-                specID: 'specID', // Can consume a live spec ID OR the last published ID
+                specID: 'specID', // live spec ID
```

Comment:
`connectorID` and `specID` diverge from camel casing standard.

Additionally, are you opposed to renaming the `specId` property to `liveSpecId`? Or the renaming the constant that reads this value (e.g., _src/components/collection/Config.tsx_) to `liveSpecId`?